source/code/source/playfab/PlayFabCurlHttpPlugin.cpp

```diff
@@ -164,31 +164,31 @@ namespace PlayFab
         // Set up headers
         curl_slist* curlHttpHeaders = nullptr;
 
-        // if(!TryCurlAddHeader(std::move(requestContainer), curlHttpHeaders, "Accept: application/json"))
-        // {
-        //     return;
-        // }
+        if(!TryCurlAddHeader(std::move(requestContainer), curlHttpHeaders, "Accept: application/json"))
+        {
+            return;
+        }
 
 
-        // if(!TryCurlAddHeader(std::move(requestContainer), curlHttpHeaders, "Content-Type: application/json; charset=utf-8"))
-        // {
-        //     return;
-        // }
+        if(!TryCurlAddHeader(std::move(requestContainer), curlHttpHeaders, "Content-Type: application/json; charset=utf-8"))
+        {
+            return;
+        }
 
-        // if(!TryCurlAddHeader(std::move(requestContainer), curlHttpHeaders, ("X-PlayFabSDK: " + PlayFabSettings::versionString).c_str()))
-        // {
-        //     return;
-        // }
+        if(!TryCurlAddHeader(std::move(requestContainer), curlHttpHeaders, ("X-PlayFabSDK: " + PlayFabSettings::versionString).c_str()))
+        {
+            return;
+        }
 
-        // if(!TryCurlAddHeader(std::move(requestContainer), curlHttpHeaders, "X-ReportErrorAsSuccess: true"))
-        // {
-        //     return;
-        // }
+        if(!TryCurlAddHeader(std::move(requestContainer), curlHttpHeaders, "X-ReportErrorAsSuccess: true"))
+        {
+            return;
+        }
 
-        curlHttpHeaders = curl_slist_append(curlHttpHeaders, "Accept: application/json");
-        curlHttpHeaders = curl_slist_append(curlHttpHeaders, "Content-Type: application/json; charset=utf-8");
-        curlHttpHeaders = curl_slist_append(curlHttpHeaders, ("X-PlayFabSDK: " + PlayFabSettings::versionString).c_str());
-        curlHttpHeaders = curl_slist_append(curlHttpHeaders, "X-ReportErrorAsSuccess: true");
+        // curlHttpHeaders = curl_slist_append(curlHttpHeaders, "Accept: application/json");
+        // curlHttpHeaders = curl_slist_append(curlHttpHeaders, "Content-Type: application/json; charset=utf-8");
+        // curlHttpHeaders = curl_slist_append(curlHttpHeaders, ("X-PlayFabSDK: " + PlayFabSettings::versionString).c_str());
+        // curlHttpHeaders = curl_slist_append(curlHttpHeaders, "X-ReportErrorAsSuccess: true");
```

Comment:
Let's do a full test pass across all platforms, and be really certain about this change.
After that, let's remove these code comments, as we've clearly already been down this road.
VERY rigorous testing is needed.